transfers/file-transfers.md

```diff
@@ -99,19 +110,35 @@ Or
 
 * /ACQ_1234_123456
   * /metadata
-  * /objects
+  * /objects-->
+
+### Transfer files in office with transfer_rsync.py
+
+{: .important}
+Files that have been updated by the donor within the past 30 days should be quarantined for 30 days to ensure that all virus definitions are up to date.
+
+Files are transferred from digital carriers in office using the [transfer_rsync script](https://nypl.github.io/digarch/tools/working-scripts.html#transfer_rsyncpy){:target="_blank"}. This script uses rsync, a command utility used for copying and syncing file locations, to initiate file transfer then packages the files in our digital archives package structure. 
 
-## Transfer files from media object
+More information about rsync can be found on the [rsync documentation page](https://linux.die.net/man/1/rsync) and [installation instructions](../software#rsync) can be found in our software section. More information about [transfer_rsync.py](https://nypl.github.io/digarch/tools/working-scripts.html#transfer_rsyncpy){:target="_blank"} can be found in the [working scripts](https://nypl.github.io/digarch/tools/working-scripts){:target="_blank"} section of our documentation.
 
```

Comment:
Add a heading here. The instructions are buried and difficult to find.

Reply:
When you say add a heading do you mean before the rsync explananation? Is there too much linking to other pages before the actual instructions start? Perhaps turning the little rsync paragraph into a note callout could work?

Reply:
I meant to add a heading on line 123 after the 2 paragraph intro. A callout might work.